samples/python_interop/generating_n_random_bits/src/GenerateRandomNumbers.qs

```diff
@@ -0,0 +1,25 @@
+namespace GenerateRandom {
+    open Microsoft.Quantum.Arrays;
+    open Microsoft.Quantum.Convert;
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Random;
+
+    operation GenerateRandomNumbers(min: Int, max: Int) : (Result[], Int) {
```

Comment:
This code seems rather more complicated than the original issue intended it to be... The core logic is Q# generating N random bits and returning an array of them; to do this, you just need to pass a single parameter, N. Defining two parameters and using one of them to define the number of bits and not using the other one at all looks very odd. If you're generating an integer in range, then you need to do the recursion, which makes the code even more convoluted.

I would recommend to simplify this quite a bit: pass a single parameter N and just generate that many bits. 

I like returning a tuple, since it shows how to work with more complicated returns as well, but we can just say that we return the integer representation of the generated bits as well

Reply:
I returned the tuple to use the result in the Python program to count the 1s and print the integer representation of the generated bits. Should I just return the result array as suggested in the issue instead?

Reply:
It's fine with a tuple of return values, just need to make sure the descriptions match